hangman/lib/hangman/server.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Hangman.Server do
+  use GenServer
+
+  alias Hangman.Game, as: Game
+
+  def handle_call({:new_game, word}, _from, _) do
+    Game.new_game(word) |>
+    reply_game
+  end
+
+  def handle_call({:new_game}, _from, _) do
+    Game.new_game |>
+    reply_game
+  end
+
```

Comment:
# -5

So you store the game in the state, but then never actually use it. Instead, you keep passing it out to the client.

The idea of the server was to keep the internal state away from the client.